src/app/Scenes/CompleteMyProfile/CompleteMyProfile.tsx

```diff
@@ -0,0 +1,68 @@
+import { NavigationContainer, NavigationProp } from "@react-navigation/native"
+import { createStackNavigator } from "@react-navigation/stack"
+import { AvatarStep } from "app/Scenes/CompleteMyProfile/AvatarStep"
+import { ChangesSummary } from "app/Scenes/CompleteMyProfile/ChangesSummary"
+import {
+  CompleteMyProfileProvider,
+  useCompleteMyProfileContext,
+} from "app/Scenes/CompleteMyProfile/CompleteMyProfileProvider"
+import { IdentityVerificationStep } from "app/Scenes/CompleteMyProfile/IdentityVerificationStep"
+import { LocationStep } from "app/Scenes/CompleteMyProfile/LocationStep"
+import { ProfessionStep } from "app/Scenes/CompleteMyProfile/ProfessionStep"
+import { LocationWithDetails } from "app/utils/googleMaps"
+
+export type CompleteMyProfileNavigationStack = NavigationProp<CompleteMyProfileNavigationRoutes>
+
+export type CompleteMyProfileNavigationRoutes = {
+  LocationStep: undefined
+  ProfessionStep: undefined
+  AvatarStep: undefined
+  IdentityVerificationStep: undefined
+  ChangesSummary: undefined
+}
+
+export type Routes = keyof CompleteMyProfileNavigationRoutes
+
+export type NavigationPayloadField = {
+  field: string
+  value: string | boolean | Partial<LocationWithDetails>
+}
+
+const Stack = createStackNavigator()
+
+const CompleteMyProfileNavigator = () => {
+  const { steps } = useCompleteMyProfileContext()
+
+  return (
+    <NavigationContainer independent>
+      <Stack.Navigator screenOptions={{ headerShown: false }}>
```

Comment:
did you consider having the header here? it should help you make the transition smoother between the screens by keeping the header
```suggestion
    <NavigationContainer independent>
      <Header />
      <Stack.Navigator screenOptions={{ headerShown: false }}>
```

https://github.com/artsy/eigen/assets/11945712/cf87cfd4-7b34-4d0e-b023-74e1778d0bea

Reply:
After some testing, I noticed some jankiness when using it(tried some perf improvs like useCallback for the header and stuff, same result):

https://github.com/artsy/eigen/assets/15792853/304e060b-aa4f-4c0c-b16d-383e84409ee7

Reply:
It looks better now, there's some minor padding to adjust when having the keyboard open in Android but should be good to go I believe(we'll address more stuff during QA too)

https://github.com/artsy/eigen/assets/15792853/87564dc3-975b-4cde-8346-8516457a8cc3

Reply:
awesome!

Reply:
Btw, we faced the same issue of the keyboard in the submit artwork flow and we had to wrap the bottom buttons with a special container to solve it. 
https://github.com/artsy/eigen/blob/f32e39db641275331c084ff41da2f9dff7ec0015/src/app/Scenes/SellWithArtsy/ArtworkForm/Components/SubmitArtworkBottomNavigation.tsx#L192